test/compare/test_dm_tree.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import tree
```

Comment:
add comments for preventing confusing, such as:
```python
import tree  # dm-tree https://github.com/deepmind/tree
```

Reply:
Completed.